src/mumble/CMakeLists.txt

```diff
@@ -655,6 +656,9 @@ else()
 		if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 			find_library(LIB_RT rt)
 			target_link_libraries(mumble_client_object_lib PUBLIC ${LIB_RT})
+			target_sources(mumble_client_object_lib PRIVATE "IPCUtils_linux.cpp")
+		else()
+			target_sources(mumble_client_object_lib PRIVATE "IPCUtils_unix.cpp")
```

Comment:
This differentiation should be solved with ifdefs in a single implementation. Using two different source files for this and using cmake to "ifdef" the correct one in, seems overkill.